docs/source/configuration.rst

```diff
@@ -602,7 +602,8 @@ debug
 
 * Default:  ``false``
 
-If set to ``true``, debugging output will be logged to the browser console.
+If set to ``true``, debugging output will be logged to the browser console. Additionally the ``/inject`` command will be enabled, which allows to inject stanzas for debugging
```

Comment:
allows *you* to

Reply:
The code only supports injecting `<message>` stanzas, which is not made clear here.

Reply:
Updated the documentation. Hope it's clearer now.